Formula/typedb.rb

```diff
@@ -1,9 +1,10 @@
 class Typedb < Formula
-  desc "Distributed hyper-relational database for knowledge engineering"
+  desc "Strongly-typed database with a rich and logical type system"
   homepage "https://vaticle.com/"
   url "https://github.com/vaticle/typedb/releases/download/2.14.3/typedb-all-mac-2.14.3.zip"
   sha256 "41a574d4d0fafcdfd678599b488dfb3aa7e2c4664e111dbd479bdf0a4dbd12a7"
   license "AGPL-3.0-or-later"
+  revision 1
```

Comment:
I think it's better not to do this since it'll trigger an update in which people will still lose their data. Would be better to wait for the next version update.